pkg/integration/dockerCmd_utils_test.go

```diff
@@ -389,7 +387,7 @@ func TestHelperProcess(t *testing.T) {
 			fmt.Fprintf(os.Stderr, "an error has occurred")
 			os.Exit(1)
 		case "a command that times out":
-			time.Sleep(10 * time.Millisecond)
+			time.Sleep(30 * time.Millisecond)
```

Comment:
This is used there : https://github.com/docker/docker/blob/master/pkg/integration/dockerCmd_utils_test.go#L190.

Basically, `TestDockerCmdWithTimeout`, will call the fake `dockerBinary` command that runs the test binary with special flags and gets to there : `TestHelperProcess` (This is basically what's done in the golang exec tests :smile_cat:). The idea behind that case is that if I do some `exec.Command(dockerBinary, "a", "command", "that", "times", "out")`, I want a long running command so that it will be killed by my test case. And I'm checking afterwards that it has failed — although you make a point.. I should change the `os.Exit(0)` to something else to make sure it did not complete (and then didn't get a different exit code).

And also, I don't remember why I did change `10` to `30` :sweat:. Guess it's from my early tests on this racey thingy.